apps/browser/qml/pages/DownloadPickerPage.qml

```diff
@@ -0,0 +1,13 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Pickers 1.0
+
+FolderPickerPage {
```

Comment:
This page looks redundant to me: you can connect directly to `onSelectedPathChanged` in the BrowserPage.qml code to avoid the need for the additional `selected` signal (and then use `page.selectedPath` instead of passing `path`.